packages/jest-resolve-dependencies/src/index.ts

```diff
@@ -98,11 +98,13 @@ class DependencyResolver {
           }, []),
         );
       }
-      return result.concat(Array.from(related).map(file => ({file})));
+      return result.concat(
+        Array.from(related).map(file => ({dependencies: [], file})),
```

Comment:
Easier to just disallow `dependencies` being `undefined` in `ResolvedModule` to avoid `undefined` checks

Reply:
Yeah!